packer/linux/scripts/install-buildkite-test-splitter.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+case $(uname -m) in
+x86_64) ARCH=amd64 ;;
+aarch64) ARCH=arm64 ;;
+*) ARCH=unknown ;;
+esac
+
+echo "Creating test-splitter user and group..."
+sudo useradd --base-dir /var/lib --uid 2000 test-splitter
+sudo usermod -a -G docker test-splitter
+
+TEST_SPLITTER_VERSION=0.7.2
+echo "Downloading test-splitter v${TEST_SPLITTER_VERSION}..."
+sudo curl -Lsf -o /usr/bin/test-splitter \
+  "https://github.com/buildkite/test-splitter/releases/download/v0.7.2/test-splitter-linux-${ARCH}"
```

Comment:
Are we happy to continue using github.com releases for this build step, or should we add a step to the test-splitter release pipeline to upload to s3 (download.buildkite.com) like the agent.

Reply:
I think there's a whole project in making all the various places (GitHub, S3, Buildkite Packages?) and packaging formats (raw, .zip, .deb, .rpm. ...) we provide binaries in consistent across all our projects. 

Since this only needs to work when packer builds the AMI, I think just GitHub is fine for now?